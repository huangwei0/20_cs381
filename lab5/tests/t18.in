findall(Z, child(Z, 'Prince Albert'), X), 
print(X), 
print('\n'), 
halt.
