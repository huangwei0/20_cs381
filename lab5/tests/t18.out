[King Edward VII,Prince Alfred: Duke of Edinburgh,Prince Arthur,Prince Leopold,Victoria: Princess Royal,Princess Alice,Princess Helena,Princess Louise,Princess Beatrice]
